content/master/concepts/composition-functions.md

```diff
@@ -1,1026 +1,699 @@
+template Crossplane resources. Crossplane calls composition functions to
+determine what resources it should create when you create a composite resource
+(XR). You can write a function to template resources using a general purpose
+programming language like Go or Python. Using a general purpose programming
+language allows a Function to use more advanced logic to template resources,
+like loops and conditionals.
 
-Composition Functions allow you to supplement or replace your Compositions with
-advanced logic not implementable through available patching strategies. 
+You can build a function using general purpose programming languages such as Go
+or Python. The Crossplane community has also built functions that let you
+template Crossplane resources using [CUE](https://cuelang.org), Helm-like
+[Go templates](https://pkg.go.dev/text/template) or
+[Patch and Transforms]({{<ref "./patch-and-transform">}}).
 
+## Install a composition function
 
-You can build a Function using general-purpose programming
-languages such as Go or Python, or relevant tools such as Helm, 
-[Kustomize](https://kustomize.io/), or
-[CUE](https://cuelang.org/).  
+Installing a Function creates a function pod. Crossplane sends requests to this
+pod to ask it what resources to create when you create a composite resource.
 
-Functions complement contemporary "Patch and Transform" (P&T) style
-Composition. It's possible to use only P&T, only Functions, or a mix of both in
-the same Composition.
+Install a Function with a Crossplane
+{{<hover label="install" line="2">}}Function{{</hover >}} object setting the
+{{<hover label="install" line="6">}}spec.package{{</hover >}} value to the
+location of the function package.
 
-```yaml
-apiVersion: apiextensions.crossplane.io/v1
-kind: Composition
+
+For example, to install [Function Patch and Transform](https://github.com/crossplane-contrib/function-patch-and-transform),
+
+```yaml {label="install"}
+apiVersion: pkg.crossplane.io/v1beta1
+kind: Function
 metadata:
-  name: example
+  name: function-patch-and-transform
 spec:
-  compositeTypeRef:
-    apiVersion: database.example.org/v1alpha1
-    kind: XPostgreSQLInstance
-  functions:
-  - name: my-cool-Function
-    type: Container
-    container:
-      image: xpkg.upbound.io/my-cool-Function:0.1.0
+  package: xpkg.upbound.io/crossplane-contrib/function-patch-and-transform:v0.1.4
 ```
 
-A Composition Function is a short-lived OCI container that tells Crossplane how
-to reconcile a Composite Resource (XR). The preceding example shows a minimal
-`Composition` that uses a Composition Function. Note that it has a `functions`
-array rather than the typical P&T style array of `resources`.
+{{< hint "tip" >}}
+Functions are Crossplane Packages. Read more about Packages in the
+[Packages documentation]({{<ref "packages" >}}).
+{{< /hint >}}
 
-## Enabling functions
+By default, the Function pod installs in the same namespace as Crossplane
```

Comment:
No. There's a Crossplane flag `--namespace` that overrides it.

FWIW I copied this section verbatim from the existing Providers documentation. +1 to explaining how this can be overridden but I'd prefer to do that in a separate PR once https://github.com/crossplane/docs/pull/566 has landed, since I imagine there will be more updates.